spec/system/kudo_system_spec.rb

```diff
@@ -24,6 +25,7 @@
     fill_in 'kudo[title]', with: new_kudo.title
     fill_in 'kudo[content]', with: new_kudo.content
     select employee.email
+    select company_value.title
```

Comment:
I think we should also consider testing giving kudo without a `company_value`

Reply:
I added this new short test in Kudo spec. Also, I corrected all the specs (don't be scared :)) due to Oscar's comment from the previous PR.